File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRoot/IfcRelationship/IfcRelConnects/IfcRelConnectsElements/IfcRelConnectsElements.cs
public class IfcRelConnectsElements : IfcRelConnects
{
    public IfcConnectionGeometry ConnectionGeometry { get; set; }
    public IfcElement RelatingElement { get; set; }
    public IfcElement RelatedElement { get; set; }

    // === EXTENDED BY ===
    // IfcRelConnectsPathElements
    // IfcRelConnectsWithRealizingElements

    // === WHERE CLAUSES ===
    // NoSelfReference : RelatingElement :<>: RelatedElement
}

File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRoot/IfcRelationship/IfcRelConnects/IfcRelConnectsElements/IfcRelConnectsElements.vb
Public Class IfcRelConnectsElements Inherits IfcRelConnects
    Public ConnectionGeometry As IfcConnectionGeometry
    Public RelatingElement As IfcElement
    Public RelatedElement As IfcElement

    ' === EXTENDED BY ===
    ' IfcRelConnectsPathElements
    ' IfcRelConnectsWithRealizingElements

    ' === WHERE CLAUSES ===
    ' NoSelfReference : RelatingElement :<>: RelatedElement
End Class
